### PR TITLE
chore(ci): cache .build swift dir

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

The `.build` folder ca save a lot of time when cached properly

### CODE REVIEW

File: .github/actions/setup/action.yml [hunk 1]
```diff
@@ -202,6 +202,14 @@ runs:
       with:
         swift-version: ${{ steps.versions.outputs.SWIFT_VERSION }}
 
+    - name: Cache the build folder
+      id: cache-swift-build
+      if: ${{ inputs.language == 'swift' }}
+      uses: actions/cache@v4
+      with:
+        path: tests/output/swift/.build
+        key: swift-cts-build-${{ steps.versions.outputs.SWIFT_VERSION }}-${{ runner.os }}
```
Comment (shortcuts) [inline .github/actions/setup/action.yml:211]: should we hash the source folder maybe to invalidate it?

Comment (millotp) [reply .github/actions/setup/action.yml]: the swift build system already does that, it only recompiles changed files

Comment (shortcuts) [reply .github/actions/setup/action.yml]: ah ok then cooool

**Review by shortcuts — APPROVED** [review]
w o n d e r f u l

**Pull request merged**